mix.exs

```diff
@@ -26,10 +26,11 @@ defmodule PaperTrail.Mixfile do
 
   defp deps do
     [
-      {:ecto, ">= 2.1.0"},
-      {:poison, ">= 3.1.0 or >= 2.0.0"},
+      {:ecto, "~> 3.0-rc", override: true},
+      {:ecto_sql, "~> 3.0-rc", override: true},
       {:ex_doc, ">= 0.0.0", only: :dev},
-      {:postgrex, "~> 0.13.0"}
+      {:postgrex, ">= 0.0.0-rc"},
```

Comment:
0.0.0-rc ?

Reply:
I think I just copied it and pasted from mix.exs of generated phoenix 1.4.0-rc app. Do you want me to change it to the latest version or "=> 0.0.0"?